src/lib/utils/hooks/SynapseAPI/user/useFavorites.ts

```diff
@@ -8,8 +8,14 @@ import { SynapseClient } from '../../..'
 import { SynapseClientError } from '../../../SynapseClientError'
 import { useSynapseContext } from '../../../SynapseContext'
 import { EntityHeader, PaginatedResults } from '../../../synapseTypes'
+import {
+  FavoriteSortBy,
+  FavoriteSortDirection,
+} from '../../../synapseTypes/FavoriteSortBy'
 
 export function useGetFavorites(
+  sort: FavoriteSortBy = 'FAVORITED_ON',
+  sortDirection: FavoriteSortDirection = 'DESC',
```

Comment:
This is the sort that's currently used on the Favorites page. It 
 * has its own client-side sorting and
 * has no column for the "favorited on" date, (not returned by the backend)

Thus a user cannot revert the sort to be the "favorited on" datetime, so I didn't want to change it.